### ISSUE TITLE
False positive for unused argument when argument is scoped

### ISSUE THREAD

**Issue by jtreher** [cflint/CFLint#227]
It looks like there is a //TODO on the code for flagging a scoped argument as used. Do you happen to know what that status of this is? It creates a lot of false positives at the moment. Thanks!

https://github.com/cflint/CFLint/blob/dev/src/main/java/com/cflint/plugins/core/UnusedArgumentChecker.java

```protected void useIdentifier(final CFIdentifier identifier) {
		String name = identifier.getName().toLowerCase();
		if (name.equals("arguments")) {
			name = identifier.Decompile(0).toLowerCase().replace("arguments.", ""); // TODO
																					// better
																					// way
																					// of
																					// doing
																					// this?
		}
		if (methodArguments.get(name) != null) {
			methodArguments.put(name, true);
		}
	}```

**Comment by ryaneberly** [comment]
fixed in dev.  Thanks!
